### ISSUE TITLE
node package needs updated in npm

### ISSUE THREAD

**Issue by opatomic** [novnc/websockify#360]
The source code in websockify.js was updated to support changes to the "ws" package. See https://github.com/novnc/websockify/issues/280. However, the package at npm has not yet been updated causing it to crash with most recent version of ws package.

**Comment by CendioOssman** [comment]
Right. We should probably do the discussed split in to separate repos first though. Hopefully soon-ish.

**Comment by YoeriVD** [comment]
Any update on this? :-)

**Comment by CendioOssman** [comment]
Not really. Way too much other work right now. Hopefully I can get back to it soon.
